Updating MKS x CDA documentation (#7320)
ovh/docs: pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-cloud-disk-array/guide.fr-fr.md

By using Public Cloud Gateway through our Managed Kubernetes Service, Public IPs on nodes are only for management purposes: [MKS Known Limits](/pages/public_cloud/containers_orchestration/managed_kubernetes/known-limits)

You can get your OVHcloud Internet Gateway's Public IP by navigating through the OVHcloud Control Panel:

`Public Cloud`{.action} > Select your tenant > `Network / Gateway`{.action} > `Public IP`{.action}

You can also get your OVHcloud Internet Gateway's Public IP by using our APIs:

> [!api]
>
> @api {v1} /cloud GET  /cloud/project/{serviceName}/region/{regionName}/gateway/{id}
>

You can find more details about how to use OVHcloud APIs with this guide: [First Steps with the OVHcloud APIs](/pages/manage_and_operate/api/first-steps)

If you want to use your Kubernetes cluster to know your Gateway Public's IP, you can run these commands:

```bash
kubectl run get-gateway-ip --image=ubuntu:latest -i --tty --rm 
```

This command will create a temporary pod and open a console.

You may have to wait a bit to let the pod be created. Once the shell appears, you can run this command:

```bash

> apt update && apt upgrade -y && apt install -y curl && curl ifconfig.me

Reviewer: ```suggestion
apt update && apt upgrade -y && apt install -y curl && curl ifconfig.ovh
```
Small suggestion for the ifconfig service :)

```

> The Public IP of the Gateway you're using should appear.

Reviewer: ```suggestion
This command will output the Public IP of the Gateway of your kubernetes cluster.
```

exit
```

Generate the URL to open in your broswer:

```bash
$ URL=$(echo "http://localhost:8001/api/v1/namespaces/default/pods/http:$FIRST_POD:/proxy/")
echo $URL
```

> You can open the URL which is displayed to access the Nginx Service.

Reviewer: ```suggestion
You can open the displayed URL to access the Nginx Service.
```

> Now let’s try to see if the data is shared with the second pod (if you have more than one node deployed).

Reviewer: ```suggestion
Use the following command to validate that the filesystem is shared with the second pod (given that you have more than one node deployed).
```

```bash
$ SECOND_POD=$(kubectl get pod -l name=nginx --no-headers=true -o custom-columns=:metadata.name | head -2 | tail -1)
URL2=$(echo "http://localhost:8001/api/v1/namespaces/default/pods/http:$SECOND_POD:/proxy/")
echo $URL2
```

Let’s try to access our new web page:

```bash
kubectl proxy
```

> And open both URLs given by the commands above to see if the data is shared with all the pods connected to the Ceph volume.

Reviewer: ```suggestion
Open both URLs given by the commands above to see if the data is shared with all the pods connected to the Ceph volume.
```
